frontend/src/component/admin/instance-privacy/index.tsx

```diff
@@ -0,0 +1,151 @@
+import { InstancePrivacy } from './InstancePrivacy';
+import { useTelemetry } from 'hooks/api/getters/useTelemetry/useTelemetry';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+interface IFeatureActivenessManagementInfo {
+    enabled: IActivenessManagementInfo;
+    disabled: IActivenessManagementInfo;
+}
+
+interface IActivenessManagementInfo {
+    environmentVariables: String[];
+    changeInfoText: String;
+}
+
+const StyledBox = styled(Box)(({ theme }) => ({
+    display: 'grid',
+    gap: theme.spacing(4),
+}));
+
+const versionCollectionDetails = {
+    title: 'Version data collection',
+    infoText:
+        "We collect the version of Unleash that you're using. We use this information to inform your Unleash instance of latest updates and and critical security patches.",
+
+    concreteDetails: {
+        'Instance ID': 'A unique ID generated for your instance',
+        Version: "The version of Unleash that you're using",
+    },
+};
+
+const featureCollectionDetails = {
+    title: 'Feature data collection',
+    infoText:
+        'We collect data about your instance to improve the Unleash product user experience. Also we use the data in case you need help from our support. Data collection is for internal use and is not shared with third parties outside Unleash. As we want you to be in control of your data, we will leave it up to you to allow us to collect your data.',
+    concreteDetails: {
+        'Feature toggles': 'The number of feature toggles in your instance',
+        Users: 'The number of users registered in your instance',
+        Projects: 'The number of projects in your instance',
+        'Context Fields': 'The number of custom context fields in use',
+        Groups: 'The number of groups present in your instance',
+        Roles: 'The number of custom roles defined in your instance',
+        Environments: 'The number of environments in your instance',
+        Segments: 'The number of segments defined in your instance',
+        Strategies: 'The number of strategies defined in your instance',
+        /*SAML: 'Whether or not SAML SSO is in use',
+        OIDC: 'Whether or not OIDC SSO is in use',*/
+        'Feature Exports': 'The number of feature exports performed',
+        'Feature Imports': 'The number of feature imports performed',
+        'Custom Strategies':
+            'The number of custom strategies defined in your instance',
+        'Custom Strategies In Use':
+            'The number of custom strategies that are in use by feature toggles',
+    },
+};
+
+const versionCollectionActivenessManagementTexts: IFeatureActivenessManagementInfo =
+    {
+        enabled: {
+            environmentVariables: ['CHECK_VERSION=false'],
+            changeInfoText:
+                'Version Info Collection can be disabled by setting the environment variable to anything other than true and restarting Unleash.',
```

Comment:
I'd suggest downcasing some of these words. Also: can we name the env variable in the text too for maximum explicitness? Further, "anything other than true" isn't quite true (because `t` and `1` still turn it on), I'd suggest suggesting to use `false` here and in the other places where it's relevant.

```suggestion
                'Version info collection can be disabled by setting the environment variable `CHECK_VERSION` to `false` and restarting Unleash.',
```